llvm/include/llvm/Transforms/Utils/BasicBlockUtils.h

```diff
@@ -705,6 +705,9 @@ void InvertBranch(BranchInst *PBI, IRBuilderBase &Builder);
 // Check whether the function only has simple terminator:
 // br/brcond/unreachable/ret
 bool hasOnlySimpleTerminator(const Function &F);
+
+bool isPresplitCoroSuspendExitEdge(const BasicBlock &Src,
```

Comment:
nit: Let's add some comments for this. Especially all other function in Utils.h have a lot of comments.